src/backend/dx11/src/lib.rs

```diff
@@ -3421,6 +3423,51 @@ pub struct ImageView {
     dsv_handle: Option<ComPtr<d3d11::ID3D11DepthStencilView>>,
     rodsv_handle: Option<ComPtr<d3d11::ID3D11DepthStencilView>>,
     uav_handle: Option<ComPtr<d3d11::ID3D11UnorderedAccessView>>,
+    borrowed: bool,
+}
+
+impl Clone for ImageView {
+    fn clone(&self) -> Self {
+        Self {
+            subresource: self.subresource,
+            format: self.format,
+            rtv_handle: self.rtv_handle.clone(),
+            srv_handle: self.srv_handle.clone(),
+            dsv_handle: self.dsv_handle.clone(),
+            rodsv_handle: self.rodsv_handle.clone(),
+            uav_handle: self.uav_handle.clone(),
+            borrowed: false
+        }
+    }
+}
+
+impl ImageView {
+    fn clone_borrowed(&self) -> Self {
+        unsafe {
+                Self {
+                subresource: self.subresource,
+                format: self.format,
+                rtv_handle: self.rtv_handle.as_ref().map(|p| ComPtr::from_raw(p.as_raw())),
```

Comment:
I don't understand why we are doing this? Looks like we have a `ComPtr` and we are constructing a new `ComPtr` from the same pointer without actually cloning?